src/ResourceManager/Resources/Commands.ResourceManager/Cmdlets/Implementation/Policy/NewAzurePolicyAssignment.cs

```diff
@@ -50,10 +60,25 @@ public class NewAzurePolicyAssignmentCmdlet : PolicyAssignmentCmdletBase
         /// <summary>
         /// Gets or sets the policy assignment policy definition parameter.
         /// </summary>
-        [Parameter(Mandatory = true, ValueFromPipelineByPropertyName = true, HelpMessage = "The pollicy definition object.")]
+        [Parameter(Mandatory = true, ValueFromPipelineByPropertyName = true, HelpMessage = "The policy definition object.")]
         public PSObject PolicyDefinition { get; set; }
 
         /// <summary>
+        /// Gets or sets the policy assignment policy parameters object.
+        /// </summary>
+        [Parameter(ParameterSetName = PolicyParametersObjectParameterSetName,
+            Mandatory = false, ValueFromPipelineByPropertyName = true, HelpMessage = "The policy parameters object.")]
+        public Hashtable PolicyParametersObject { get; set; }
+
+        /// <summary>
+        /// Gets or sets the policy assignment policy parameters file path or policy parameters string.
+        /// </summary>
+        [Parameter(ParameterSetName = PolicyParametersStringParameterSetName, 
+            Mandatory = false, ValueFromPipelineByPropertyName = true, HelpMessage = "The policy parameters file path or policy parameters string.")]
+        [ValidateNotNullOrEmpty]
+        public string PolicyParameters { get; set; }
```

Comment:
For 1: So the parameters in policy def are declarations and those in policy assignment are actual values. I know the names are confusing, but I have confirmed with Mike about these names.
For 2: Will do that.